cli/deno_error.rs

```diff
@@ -185,6 +186,19 @@ impl GetErrorKind for hyper::Error {
   }
 }
 
+impl GetErrorKind for ReadlineError {
+  fn kind(&self) -> ErrorKind {
+    use ReadlineError::*;
+    match self {
+      Io(_err) => unimplemented!(),
+      Eof => ErrorKind::UnexpectedEof,
```

Comment:
So the type of `_err` here is probably io::Error. Which means you can just call kind() on it.
So: `Io(err) => err.kind()`

Reply:
Ok - done